pp-scripts/pprocess.py

```diff
@@ -29,16 +29,18 @@
 
 FIO_COL = 0 # This can be 0 or 1 depending on fio --version!
 
+# Uppercase units are for power-of-2 values and lowercase units are for power-of-10 values
+# See https://fio.readthedocs.io/en/latest/fio_doc.html#units
 suffixmap = {
     'n' : 1e-9   ,
     'u' : 1e-6   ,
     'm' : 1e-3   ,
     ' ' : 1      ,
     'k' : 1e3    ,
-    'K' : 1e3    ,
-    'M' : 1e6    ,
-    'G' : 1e9    ,
-    'T' : 1e12
+    'K' : 2**3    ,
```

Comment:
@ljishen why is this not 2**10??

Reply:
How could it happen?! Please bear with me on this. Will fix it.